esp/public/media/scripts/program/modules/flag-results-page.js

```diff
@@ -59,6 +59,27 @@ function hideAll () {
     $j(".flag-detail").hide();
 }
 
+function approveAll () {
+    var r = confirm("Are you sure you'd like to approve all of these classes?");
```

Comment:
I'd allcaps "all" to make it more obvious, but otherwise I think this is enough.